File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ConfirmCodeForForgotPasswordBody.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;


public class ConfirmCodeForForgotPasswordBody {


        @SerializedName("email")
        private String email;

        @SerializedName("password")
        private String password;

        @SerializedName("confirmation_code")
        private String confirmation_code;


        public ConfirmCodeForForgotPasswordBody(String email,String confirmation_code,String password) {
            this.email = email;
            this.password = password;
            this.confirmation_code = confirmation_code;
        }

        @Override
        public String toString() {
            return "ResetPasswordBody{" +
                    "phoneNumber=" + email +
                    '}';
        }



}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/user/ConfirmCodeForForgotPasswordBody.kt
package com.lattis.data.entity.body.user

import com.google.gson.annotations.SerializedName

class ConfirmCodeForForgotPasswordBody(
    @field:SerializedName("email") private val email: String,
    @field:SerializedName("confirmation_code") private val confirmation_code: String,
    @field:SerializedName("password") private val password: String
) {

    override fun toString(): String {
        return "ResetPasswordBody{" +
                "phoneNumber=" + email +
                '}'
    }

}